packages/helm-charts/celo-fullnode/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "celo-fullnode.fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ template "celo-fullnode.name" . }}
```

Comment:
Should we use `standard.short_labels` or `standard.labels` here?

Reply:
Really the ingress is the default ingress resource. It can be fully customized with parameters, but do you think is a good idea to keep it if we do not plan to use it right now? If not we can directly remove this file (an associated values in default values.yaml).